x/programs/rust/sdk-macros/tests/ui/too-many-state-key-variants.rs

```diff
@@ -0,0 +1,14 @@
+use seq_macro::seq;
+use wasmlanche_sdk::state_keys;
+
+seq!(N in 0..255  {
+    #[state_keys]
+    pub enum StateKeysFail {
+        #(
+            Variant~N,
+        )*
+        Variant256,
+    }
+});
```

Comment:
tiny nit: this will actually not generate the `Variant255` since the range is not inclusive of this number
<img width="162" alt="Screenshot 2024-05-30 at 23 25 47" src="https://github.com/ava-labs/hypersdk/assets/51274081/c4e1d5a7-3515-4067-a452-bac4119cccca">